core/deployment/src/main/java/io/quarkus/deployment/builditem/ConfigClassBuildItem.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.arc.deployment;
+package io.quarkus.deployment.builditem;
```

Comment:
I wonder if we should just deprecate the old one for 2.2? Or we are sure it's not used anywhere in our extension ecosystem?

(deprecating a build item is easy, you just consume the old ones and produce the new ones)

Reply:
I think it is ok to rename, this is only intended for internal usage.

This raises an interesting question. Do we have a way to distinguish which build step may be used safely on the ecosystem and which are for internal only? Unfortunately, a regular access modifier is not enough.

Reply:
No, we don't.